[2024-11-30T00:59:50.543+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-30T00:59:50.560+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag1.process_task_tag2 manual__2024-11-30T00:59:43.752127+00:00 [queued]>
[2024-11-30T00:59:50.576+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag1.process_task_tag2 manual__2024-11-30T00:59:43.752127+00:00 [queued]>
[2024-11-30T00:59:50.577+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2024-11-30T00:59:50.634+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): process_task_tag2> on 2024-11-30 00:59:43.752127+00:00
[2024-11-30T00:59:50.643+0000] {logging_mixin.py:190} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=213) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-11-30T00:59:50.644+0000] {standard_task_runner.py:72} INFO - Started process 227 to run task
[2024-11-30T00:59:50.644+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'dag1', 'process_task_tag2', 'manual__2024-11-30T00:59:43.752127+00:00', '--job-id', '24', '--raw', '--subdir', 'DAGS_FOLDER/dag_data.py', '--cfg-path', '/tmp/tmpsg1ammih']
[2024-11-30T00:59:50.649+0000] {standard_task_runner.py:105} INFO - Job 24: Subtask process_task_tag2
[2024-11-30T00:59:50.849+0000] {task_command.py:467} INFO - Running <TaskInstance: dag1.process_task_tag2 manual__2024-11-30T00:59:43.752127+00:00 [running]> on host 45770b1a429e
[2024-11-30T00:59:51.090+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='admin' AIRFLOW_CTX_DAG_ID='dag1' AIRFLOW_CTX_TASK_ID='process_task_tag2' AIRFLOW_CTX_EXECUTION_DATE='2024-11-30T00:59:43.752127+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-11-30T00:59:43.752127+00:00'
[2024-11-30T00:59:51.094+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-11-30T01:09:21.985+0000] {dag_data.py:68} INFO - Toutes les données de sales.csv ont été publiées avec succès.
[2024-11-30T01:09:21.987+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-11-30T01:09:22.008+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-11-30T01:09:22.009+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=dag1, task_id=process_task_tag2, run_id=manual__2024-11-30T00:59:43.752127+00:00, execution_date=20241130T005943, start_date=20241130T005950, end_date=20241130T010922
[2024-11-30T01:09:22.123+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-11-30T01:09:22.185+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-11-30T01:09:22.191+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
